funflow-tutorial/notebooks/Tutorial1.ipynb

```diff
@@ -50,18 +50,21 @@
     "`funflow` works with a wide range of task granularities.\n",
     "A `Task` can be a simple Haskell function, a database query, a command to run in a Docker container, or more. \n",
     "\n",
-    "There are several different types of tasks in Funflow, each describing a specific type of computation. \n",
-    "Tasks are defined in the `Funflow.Tasks` subpackage.\n",
-    "The most basic task, the datatype `PureTask`, represents  a pure Haskell function which has no _side effects_ such as reading a file or running a command.\n",
+    "Accordingly, `funflow` provides several task types to support these different kinds of computations.\n",
+    "These `Task`s are defined in the `Funflow.Tasks` subpackage.\n",
+    "Of these types, `PureTask` is the simplest and represents  a Haskell function which has _no side effects_, like reading a file or running a command.\n",
     "Other task datatypes include `IOTask`, which runs a Haskell function that can perform I/O (e.g. reading a file), and `DockerTask`, which runs a \n",
     "[Docker](https://docs.docker.com/get-docker/) container.\n",
     "\n",
     "\n",
-    "## How to create Flows\n",
+    "## How to create a flow\n",
+    "A flow is most easily built either...\n",
+    "1. ...from a task (or any value of a type in `IsFlow`), using `toFlow`\n",
+    "2. ...from a Haskell function, using an appropriately typed _smart constructor_ provided in this package\n",
```

Comment:
Smart constructors also work with data which isn't exactly a function (e.g. `dockerFlow`). I think `simpleFlow` is the only one at the moment which takes a function as its input. We should probably update the language here slightly to put less emphasis on the "from a Haskell function" part and more on the smart constructor part.